### PR TITLE
feat(docs): add azure peering documentation

### PR DESCRIPTION
**Pull request by dkropachev**
**Check List**
- [ ] ~~Tests has been run in packages where changes made if available~~
- [ ] ~~Linter has been run for changed code~~
- [ ] ~~Tests for the changes have been added if not covered yet~~
- [x] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**
Add docs regarding Azure peering

### CODE REVIEW

File: docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx [hunk 1]
```diff
@@ -5,17 +5,56 @@ permalink: /cloud/configuration/connecting-with-a-vpc/azure
...
-To allow Cube Cloud to connect to a VPC on Azure, the following information is
-required:
+To allow Cube Cloud to connect to a Virtual Network on Azure, the following information is required:
+- **Virtual Network Name:** You can find it at `Virtual Networks` section on [Azure Portal][https://portal.azure.com].
+- **Tenant ID:** You can find it at `Azure Active Directory` > `Properties` > `Tenant ID` on [Azure Portal][https://portal.azure.com].
 
 ## Setup
 
-### VPC Peering
+Unfortunately Azure cross-tenant peering suppose that you grant peering role to the user id/service principal to the peering party
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx:14]: Not really sure what this sentence means 😅

File: docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx [hunk 2]
```diff
@@ -5,17 +5,56 @@ permalink: /cloud/configuration/connecting-with-a-vpc/azure
...
+
+On [https://portal.azure.com](https://portal.azure.com) go to `Azure Active Directory` > `External Identities` > `Cross-tenant access settings` > `Organizational Settings` > `+ Add Organization`
+Enter Tenant ID: `197e5263-87f4-4ce1-96c4-351b0c0c714a`
+Make sure that `B2B Collaboration` > `Inbound Access` > `Applications`, is set, or inherited so that it `Allows access`
+
+### Register Cube Cloud service principal at your organization
+
+`Cube Cloud service principal` info:
+Client ID: `0c5d0d4b-6cee-402e-9a08-e5b79f199481`
+Name: `cube-dedicated-infra-sp`
```
Comment (hassankhan) [inline docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx:27]: How does this information relate to the instructions below?

**Review by hassankhan — CHANGES_REQUESTED** [review]

File: docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx (hunk 1 above)
Comment (dkropachev) [reply docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx:14]: It is a shout-out to the worst user flow in the industry

File: docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx (hunk 2 above)
Comment (dkropachev) [reply docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx:27]: Just before user click "Accept" button he/she get to see this service principal information

**Review by hassankhan — APPROVED** [review]

**Review by hassankhan — APPROVED** [review]

**Pull request merged**